spec/controllers/concerns/idv_step_concern_spec.rb

```diff
@@ -46,6 +46,7 @@ def show
       it 'redirects to activated page' do
         get :show
 
+        expect(subject.current_user.profiles.size).to eq 0 # ?
```

Comment:
this seems wrong in this context.

Reply:
I think this test is checking that a user who has `.active_profile` returning something non-nil is redirected to the 'you're already activated' page. The mock on line 42 means that current_user.profiles will be empty, right? It'd probably be good to create the user such that it actually looks fully verified, I think?

Reply:
I think I'm missing how the mock works. `user.active_profile` calls `user.profiles.verified` which is empty, but maybe the mock is contriving a non-null value for `user.active_profile` just for purposes of this spec.